image/data.go

```diff
@@ -0,0 +1,59 @@
+package image
+
+type Images struct {
+	Count      int     `json:"count"`
+	Items      []Image `json:"items"`
+	Limit      int     `json:"limit"`
+	Offset     int     `json:"offset"`
+	TotalCount int     `json:"total_count"`
+}
+
+type NewImage struct {
+	CollectionId string  `json:"collection_id,omitempty"`
+	Filename string  `json:"filename,omitempty"`
+	License License   `json:"license,omitempty"`
+	Type string  `json:"type,omitempty"`
+}
+
+type License struct {
+	Title string  `json:"title,omitempty"`
+	Href string  `json:"href,omitempty"`
+}
+
+type Image struct {
+	Id string  `json:"id,omitempty"`
+	CollectionId	 string  `json:"collection_id,omitempty"`
+	State string  `json:"state,omitempty"`
+	//enum:
+	//- created
+	//- uploaded
+	//- publishing
+	//- published
+	//- deleted
+	Filename string  `json:"filename,omitempty"`
+	License License   `json:"license,omitempty"`
+	Upload ImageUpload `json:"upload,omitempty"`
+	Type string  `json:"type,omitempty"`
+	Downloads ImageDownloads  `json:"downloads,omitempty"`
+}
+
+type ImageUpload struct {
+	Path string  `json:"path,omitempty"`
+}
+
+type ImageDownloads struct {
+	Png ImagedDownloadsVariant  `json:"png,omitempty"`
+}
+
+type ImagedDownloadsVariant struct {
+	Size1920x1080 ImageDownload  `json:"1920x1080,omitempty"`
+	Size1280x720 ImageDownload  `json:" 1280x720,omitempty"`
```

Comment:
NB. This conforms to the swagger spec, but I'm not convinced this is the right approach. We need to discuss the keys for image variants in more detail.

Reply:
Maybe we can use a generic nested map in ImageDownloads, to allow for generic extensions and resolutions (or some other description like `thumbnail`)?
Unless we define a rigid set of possible keys, but I think we don't have a clear use case yet.